setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+with open(path.join(here, 'requirements.txt')) as f:
+    requirements = [r.rstrip().split('#')[0].rstrip() for r in f]
```

Comment:
To be more explicit, I'd have named this variable something to the following effect. Also, using `pathlib` as supported cross-platform i.e. Windows, MacOS, Linux.

```suggestion
from setuptools import setup, find_packages
from pathlib import Path

package_root_directory = Path(__file__).resolve().parents[0]

with open(Path(package_root_directory, 'requirements.txt')) as f:
    requirements = [r.rstrip().split('#')[0].rstrip() for r in f]
```